packages/core/src/dataTypes/enumerate/getPartialControl.ts

```diff
@@ -14,6 +14,9 @@ export function getPartialControl(
   return {
     type: "String",
     allowNull: props?.required == null ? props?.required : !props.required,
+    allowNullInfer: (props?.required == null
+      ? props?.required
+      : !props.required) as boolean,
```

Comment:
`enumerate` didn't get the generic added since it's not required for us to jump over from v1->v2 on frontend, I am just trying to add minimal hacks to get feature parity between the 2 versions so we can get all synced up!